applications/ecallmgr/src/ecallmgr_fs_authz.erl

```diff
@@ -83,6 +84,13 @@ maybe_authorize_channel(Props, Node) ->
             end
     end.
 
+-spec is_referred(wh_proplist()) -> 'true' | 'false'.
+is_referred(Props) ->
+  case kzd_freeswitch:ccv(Props, [<<"Referred-By">>, <<"Referred-To">>]) of
```

Comment:
I'm confused how this can ever return a two-tuple? Did you mean to put those in separate calls to `kzd_freeswitch:ccv/2`?